src/main/resources/templates.conf

```diff
@@ -33,7 +33,12 @@ templatecontrol {
     play-scala-tls-example,
     play-scala-websocket-example,
 
-    play-webgoat
+    play-webgoat,
+
+    # TemplateControl is too agressive when applied on g8 templates as it replaces
+    # all g8 variables. Current workaround is to revert those changes manually
+    play-java-seed.g8,
```

Comment:
Which changes do we need to revert manually?

It could be good to know these better, see if we can avoid them, and not add more manual steps on the release process.

Reply:
Yep, I totally agree. I don't think we should stop here and accept manual tweaks. 

Once template control makes the PR, one needs to treat the seeds PRs with extra care and revert all changes that touched variables. 

For instance, after running template control we get...
```
def playVersion = "2.6.19"
```
this should be reverted to 
```
def playVersion = '$play_version$'
```

Basically, the manual intervention is: take the PR branch, revert all variable replacements and update the PR. That's terrible stuff. Error-prone and time-consuming.

I have opened an issue to explore the possible strategies for that. 
https://github.com/lightbend/templatecontrol/issues/62